### DISCUSSION TITLE
Default UV2 table

### DISCUSSION THREAD

**Discussion by ChanseyIsTheBest** [hanai3Bi/Switch-OC-Suite#40] [Ram Timing | Customization]
Your automatically calculated UV2 table for Mariko depends on your GPU speedo. You can also lower your UV by making a custom table which is UV3. You can check GPU volt in overlay under Misc.
I want to know how much variation there is. (i.e. 1570 Speedo = 1276 GPU =  835mv)
My GPU Speedo = 1688
844 and below - 610 mv
921 - 620 mv
998 - 655 mv
1075 - 685 mv
1152 - 710 mv
1228 - 755 mv
1267 - 780 mv

**Comment by hanai3Bi** [comment]
1. For gpu, temp coefficients are used, so voltage differs with temperature.
2. You can just simply calculate voltage value from dvfs table if you know speedo, with simple(?) calculation

**Reply by ChanseyIsTheBest** [reply]
Ah, my bad I didn't realize about the temperature thing. Does the voltage lower with higher temperature?

**Reply by hanai3Bi** [reply]
higher temp = higher volt
